### DISCUSSION TITLE
Keycloak Karavan realm since 4.0.0

### DISCUSSION THREAD

**Discussion by mvolf** [apache/camel-karavan#984] [Q&A] [answered]
karavan-realm.json is missing in the repo since 4.0.0 version. Is there a reason for this or it's just a mistake @mgubaidullin ?

**Comment by mgubaidullin** [comment, marked as answer]
It was deleted with other obsolete documentation and demos.
If we have enough manpower to continue sso/keycloak integration development a new realm example will be added.
